school/index.html

```diff
@@ -266,6 +283,16 @@ <h2 class="figure-heading" aria-describedby="tip-repayment-rate">
                       </figcaption>
                     </figure>
 
+                    <div class="legend-wrapper">
+                      <div class="legend-nat_average">
+                        <div class="legend-nat_average-box">
+                          <div class="legend-nat_average-box-line">
+                          </div>
+                        </div>
+                        <div class="legend-nat_average-words">National Average</div>
+                      </div>
+                    </div>
+
```

Comment:
Maybe! Tell me more -- I'm not seeing it right out of the box (get it?) (div? box?) (is that even a joke?). Also -- maybe for later, you're right.